core/src/main/resources/hudson/lifecycle/WindowsInstallerLink/_restart.jelly

```diff
@@ -21,21 +21,44 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
-
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <st:statusCode value="503" /><!-- SERVICE NOT AVAILABLE -->
-  <l:layout permission="${app.ADMINISTER}">
-    <st:include it="${app}" page="sidepanel.jelly" />
-    <l:main-panel>
-      <h1 style="margin-top:4em">
-        ${%Please wait while Jenkins is restarting}<span id="progress">...</span>
-      </h1>
-      <p style="color:gray;">
-        ${%blurb}
-      </p>
-      
-      <script>applySafeRedirector('${rootURL}/')</script>
-    </l:main-panel>
-  </l:layout>
-</j:jelly>
\ No newline at end of file
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:x="jelly:xml">
+    <j:new var="h" className="hudson.Functions" />
+    <st:statusCode value="503" /><!-- SERVICE NOT AVAILABLE -->
+    <st:header name="Expires" value="0" />
+    <st:header name="Cache-Control" value="no-cache,no-store,must-revalidate" />
+    <!-- response contentType header -->
+    <st:contentType value="text/html;charset=UTF-8" />
+    <!-- get default/common page variable -->
+    ${h.initPageVariables(context)}
```

Comment:
The content-type, initialization of `hudson.Functions`, and this line, can all be replaced by using `<l:view>` around this view.

Reply:
I will not take the risk to break something here as the goal is only to un-inline scripts / styles.